test/e2e/case5_policy_automation_test.go

```diff
@@ -135,7 +136,448 @@ var _ = Describe("Test policy automation", func() {
 				context.TODO(), "create-service-now-ticket", metav1.GetOptions{},
 			)
 			Expect(err).To(BeNil())
-			Expect(policyAutomation.Object["spec"].(map[string]interface{})["mode"]).To(Equal("disabled"))
+			Expect(
+				policyAutomation.Object["spec"].(map[string]interface{})["mode"],
+			).To(Equal(string(policyv1beta1.Disabled)))
+		})
+		// Create two events then two ansiblejobs for each event
+		It("Test mode = everyEvent without delayAfterRunSeconds", func() {
+			By("Patching policy to make both clusters back to Compliant")
+			opt := metav1.ListOptions{
+				LabelSelector: common.RootPolicyLabel + "=" + testNamespace + "." + case5PolicyName,
+			}
+			replicatedPlcList := utils.ListWithTimeout(clientHubDynamic, gvrPolicy, opt, 2, true, defaultTimeoutSeconds)
+			for _, replicatedPlc := range replicatedPlcList.Items {
+				replicatedPlc.Object["status"] = &policiesv1.PolicyStatus{
+					ComplianceState: policiesv1.Compliant,
+				}
+				_, err := clientHubDynamic.Resource(gvrPolicy).Namespace(replicatedPlc.GetNamespace()).UpdateStatus(
+					context.TODO(), &replicatedPlc, metav1.UpdateOptions{},
+				)
+				Expect(err).To(BeNil())
+			}
+			By("Waiting until the status on each cluster is updated to Compliant")
```

Comment:
Based on many times of testing on OCM env, unlike `once` mode testing, we need to verify each policy status updating is completed for `everyEvent` test case. Because it will update the policy status very often and this check will increase the testing stability.